pkg/storage/base58wrapper/base58wrapper.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package base58wrapper
+
+import (
+	"github.com/btcsuite/btcutil/base58"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// package base58wrapper offers provider.Store wrappers for stores with certain constraints on key values such as
+// reserved prefix values (eg key IDs starting with underscore) by wrapping key values in another format supported by
+// the store.
+
+// NewBase58StoreWrapper creates a new Base64ToBase58StoreWrapper of store.
+func NewBase58StoreWrapper(store storage.Store) storage.Store {
+	return &Base64ToBase58StoreWrapper{store: store}
+}
+
+// Base64ToBase58StoreWrapper is a wrapper store that converts key IDs from base64 to base58 encoded values.
+// Keys IDs will be stored in the embedded store base58 encoded while the user of this wrapper store will interact with
+// the wrapper store using key IDs base64 raw (no padding) URL encoded.
+type Base64ToBase58StoreWrapper struct {
```

Comment:
Should not mention Base64 in the name, this wrapper takes any string and converts it to base58

Reply:
I'll push an update, good catch! Since I don't use base64 decoding anymore, there's no need to mention Base64 in the name.

Reply:
renamed